resources/AppxPackaging/Manifest/Schema/Xerces/AppxManifestTypes.xsd

```diff
@@ -0,0 +1,1326 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- @@BEGIN_SDKSPLIT
+
+  !!!WARNING!!!
+
+  This schema is closed for any changes.  Any changes to this schema file
+  are moderated by the AppModel team.  Changes to the schema can potentially
+  break .appx packages and manifests as well as product and test code that
+  relies on this schema.
+
+  Please contact the 'manifest' alias if you need to request a change to
+  this schema.
+
+@@END_SDKSPLIT -->
+<!--
+  Copyright (c) Microsoft Corporation. All rights reserved.
+
+  Defines common types shared by multiple namespaces.  This schema is imported
+  into various product-specific schemas that define elements and attributes
+  based on types declared here.
+-->
+<xs:schema attributeFormDefault="unqualified"

```

Comment:
What is this file for?

Reply:
Not needed, I forgot to delete it...

---
In reply to: [273616604](https://github.com/Microsoft/msix-packaging/pull/82#discussion_r273616604) [](ancestors = 273616604)